src/core/term.ml

```diff
@@ -434,6 +438,7 @@ let cmp : term cmp =
     | TEnv(e,ts), TEnv(e',ts') ->
         lex cmp_tenv (Array.cmp cmp) (e,ts) (e',ts')
     | TRef r, TRef r' -> Stdlib.compare r r'
+    | Plac _, Plac _ -> 0
```

Comment:
I am not sure about that but I would prefer to have `Stdlib.compare t t'` instead of `0`.